pgoapi/pgoapi.py

```diff
@@ -529,9 +529,20 @@ def attempt_catch(self, encounter_id, spawn_point_id, capture_probability=None):
             capture_probability = {}
         # Max 4 attempts to catch pokemon
         while catch_status != 1 and self.inventory.can_attempt_catch() and catch_attempts < 11:
+            item_capture_mult = 1.0
+
+            # Try to use a berry to increase the chance of catching the pokemon when we have failed enough attempts
+            if catch_attempts > self.config.get("CAPTURE", {}).get("MIN_FAILED_ATTEMPTS_BEFORE_USING_BERRY", 1) and self.inventory.has_berry():
+                self.log.info("Feeding da razz berry!")
+                r = self.use_item_capture(item_id=self.inventory.take_berry(), encounter_id=encounter_id, spawn_point_id=spawn_point_id).call()['responses']['USE_ITEM_CAPTURE']
+                if "success" in r:
```

Comment:
if success in r and success is true , i think its a boolean value. 

https://github.com/AeonLucid/POGOProtos/blob/master/src/POGOProtos/Networking/Responses/UseItemCaptureResponse.proto